<launch>
  <node name="radar_node_k79_gps" pkg="radar_ros_interface" type="radar_node_k79_gps" args="--ip 10.0.0.75 --port 1024" output="screen" required="true" />
</launch>
